### PR TITLE
[Docs-1290]  Document old public api as deprecated

### PR DESCRIPTION
**Pull request by Russ-Urquhart-ddl**
Jira:
https://dominodatalab.atlassian.net/browse/DOCS-1290

Deprecated the old endpoints and provided links to new public endpoints.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -3094,8 +3088,7 @@ These endpoints allow you interact with your
 
 ### Get a list of user's projects [GET]
 
-**This is a beta endpoint available in Domino 2.9+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_project_by_id).**
```
Comment (noahjax) [inline apiary.apib]: This should be the endpoint for retrieving a list of projects, not getting a single project.

File: apiary.apib [hunk 2]
```diff
@@ -222,6 +222,8 @@ These endpoints allow you to interact with your environments.
 
 ### Get a list of v2 environments [GET]
 
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_environments_visible_to_a_user).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_environments_visible_to_a_user).**
```

File: apiary.apib [hunk 3]
```diff
@@ -2911,8 +2913,7 @@ These endpoints provide information about users in your deployment.
 
 ### Get a list of users [GET]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_all_users_visible_to_the_current_user).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_all_users_visible_to_the_current_user).**
```

File: apiary.apib [hunk 4]
```diff
@@ -2935,8 +2936,7 @@ These endpoints provide information about users in your deployment.
 
 ### Get current user [GET]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_the_current_user).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_the_current_user).**
```

File: apiary.apib [hunk 5]
```diff
@@ -2959,9 +2959,7 @@ your deployment.
 
 ### Get a list of organizations [GET]
 
-**This endpoint requires admin permissions.**<br>
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_all_organizations_only_accessible_to_admin_users).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_all_organizations_only_accessible_to_admin_users).**
```

File: apiary.apib [hunk 6]
```diff
@@ -2982,8 +2980,7 @@ your deployment.
 
 ### Create a new organization [POST]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_create_an_organization).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_create_an_organization).**
```

File: apiary.apib [hunk 7]
```diff
@@ -3021,8 +3018,7 @@ your deployment.
 
 ### Get a list of my organizations [GET]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_the_organizations_for_a_user).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3021]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_the_organizations_for_a_user).**
```

File: apiary.apib [hunk 8]
```diff
@@ -3043,8 +3039,7 @@ your deployment.
 
 ### Get organization by ID [GET]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_an_organization_by_id.)**
```
Comment (ddl-lgaddie) [inline apiary.apib:3042]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_an_organization_by_id.)**
```

File: apiary.apib [hunk 9]
```diff
@@ -3065,8 +3060,7 @@ your deployment.
 
 ### Change the list of members [PUT]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [ADD endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_add_a_user_to_an_org) and the [DELETE endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_remove_a_user_from_an_org).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3063]:
```suggestion
**This endpoint is deprecated. For more information, see the [ADD endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_add_a_user_to_an_org) and the [DELETE endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_remove_a_user_from_an_org).**
```

File: apiary.apib (hunk 1 above)
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_project_by_id).**
```

File: apiary.apib [hunk 10]
```diff
@@ -3218,8 +3211,8 @@ These endpoints allow you interact with your
 
 ### Copy a project [POST]
 
-**This is a beta endpoint available in Domino 2.9+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_create_a_project).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3214]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_create_a_project).**
```

File: apiary.apib [hunk 11]
```diff
@@ -3482,8 +3475,7 @@ These endpoints allow you interact with your
 
 ### Add collaborator to project [POST]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_add_a_collaborator_to_this_project).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3478]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_add_a_collaborator_to_this_project).**
```

File: apiary.apib [hunk 12]
```diff
@@ -3529,8 +3521,7 @@ These endpoints allow you interact with your
 
 ### Remove collaborator from project [DELETE]
 
-**This is a beta endpoint available in Domino 2.8+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_remove_a_collaborator_from_project).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3524]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_remove_a_collaborator_from_project).**
```

File: apiary.apib [hunk 13]
```diff
@@ -3560,7 +3551,7 @@ These endpoints allow you interact with your
 
 ### Get project's Git repositories [GET]
 
-**This endpoint is available in Domino 3.4+**<br>
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_all_imported_git_repositories_in_this_project).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3554]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_all_imported_git_repositories_in_this_project).**
```

File: apiary.apib [hunk 14]
```diff
@@ -3579,7 +3570,7 @@ These endpoints allow you interact with your
 
 ### Add Git repository to project [POST]
 
-**This endpoint is available in Domino 3.4+**<br>
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_add_an_imported_git_repository_to_this_project).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3573]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_add_an_imported_git_repository_to_this_project).**
```

File: apiary.apib [hunk 15]
```diff
@@ -3633,7 +3624,7 @@ These endpoints allow you interact with your
 
 ### Remove Git repository [DELETE]
 
-**This endpoint is available in Domino 3.4+**<br>
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_remove_an_imported_repository_from_project).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3627]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_remove_an_imported_repository_from_project).**
```

File: apiary.apib [hunk 16]
```diff
@@ -3676,8 +3665,7 @@ These endpoints allow you interact with your
 
 ### Create a Dataset [POST]
 
-**This is a beta endpoint available in Domino 3.6+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_datasets_visible_to_user).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3668]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_datasets_visible_to_user).**
```

File: apiary.apib [hunk 17]
```diff
@@ -3696,8 +3684,7 @@ These endpoints allow you interact with your
 
 ### Get Dataset details [GET]
 
-**This is a beta endpoint available in Domino 3.6+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_dataset_by_id).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3687]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_dataset_by_id).**
```

File: apiary.apib [hunk 18]
```diff
@@ -3715,8 +3702,7 @@ These endpoints allow you interact with your
 
 ### Update Dataset details [PATCH]
 
-**This is a beta endpoint available in Domino 3.6+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_update_dataset_metadata).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3705]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_update_dataset_metadata).**
```

File: apiary.apib [hunk 19]
```diff
@@ -3736,9 +3722,7 @@ These endpoints allow you interact with your
 
 ### Delete Dataset [DELETE]
 
-**Note: Archive Dataset has been changed to admin only operation Delete Dataset starting Domino 4.5**<br>
-**This is a beta endpoint available in Domino 4.5+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_delete_dataset).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3725]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_delete_dataset).**
```

File: apiary.apib [hunk 20]
```diff
@@ -3761,8 +3745,7 @@ These endpoints allow you interact with your
 
 ### Get Snapshot details [GET]
 
-**This is a beta endpoint available in Domino 3.6+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_snapshot).**
```
Comment (ddl-lgaddie) [inline apiary.apib:3748]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_snapshot).**
```

File: apiary.apib [hunk 21]
```diff
@@ -3824,8 +3807,7 @@ These endpoints allow you interact with your
 
 ### Create Snapshot [POST]
 
-**This is a beta endpoint available in Domino 4.5+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_create_a_snapshot).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_create_a_snapshot).**
```

File: apiary.apib [hunk 22]
```diff
@@ -3844,8 +3826,8 @@ These endpoints allow you interact with your
 
 ### Create Dataset [POST]
 
-**This is a beta endpoint available in Domino 4.5+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_create_a_dataset).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_create_a_dataset).**
```

File: apiary.apib [hunk 23]
```diff
@@ -3867,8 +3849,7 @@ These endpoints allow you interact with your
 
 ### Add tag [POST]
 
-**This is a beta endpoint available in Domino 3.6+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_tag_a_snapshot_in_this_dataset).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_tag_a_snapshot_in_this_dataset).**
```

File: apiary.apib [hunk 24]
```diff
@@ -3890,8 +3871,7 @@ These endpoints allow you interact with your
 
 ### Remove tag [DELETE]
 
-**This is a beta endpoint available in Domino 3.6+**<br>
-**Its specification may change in future versions of Domino.**
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_remove_a_tag_from_a_dataset).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_remove_a_tag_from_a_dataset).**
```

File: apiary.apib [hunk 25]
```diff
@@ -4352,6 +4332,8 @@ Deletes an existing notification.
 
 ## Gateway Projects [/gateway/projects{?relationship,showCompleted}]
 
+**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/5.2/api_guide/8c929e/domino-public-apis/#_get_projects_visible_to_user).**
```
Comment (ddl-lgaddie) [inline apiary.apib]:
```suggestion
**This endpoint is deprecated. For more information, see the [current endpoint documentation](https://docs.dominodatalab.com/en/latest/api_guide/8c929e/domino-public-apis/#_get_projects_visible_to_user).**
```

File: apiary.apib (hunk 2 above)
Comment (ddl-lgaddie) [reply apiary.apib]: @Russ-Urquhart-ddl Let's do it this way unless we have reason to believe that the new endpoints won't be in 5.3 and up.

**Review by ddl-lgaddie — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 2 above)
Comment (Russ-Urquhart-ddl) [reply apiary.apib]: Ok, I originally had it as latest, but changed it to 5.2 as per the Readme. I can change back.

**Review by ddl-lgaddie — APPROVED** [review]

**Pull request merged**